PseudoLocalizer.Core/POProcessor.cs

```diff
@@ -23,14 +24,36 @@ public POProcessor()
         /// <param name="culture">The output culture.</param>
         public POProcessor(string culture)
         {
-            Culture = culture ?? throw new ArgumentNullException(nameof(culture));;
+            Culture = culture ?? throw new ArgumentNullException(nameof(culture));
+            ParserSettings = new POParserSettings();
+            GeneratorSettings = new POGeneratorSettings();
         }
 
         /// <summary>
         /// Gets the culture code associated with the processor.
         /// </summary>
         public string Culture { get; }
 
+#pragma warning disable CS3003 // Type is not CLS-compliant
```

Comment:
Hmm - if these types aren't CLS compliant then maybe it's better to leave them as they were as private implementation details. I'm not a fan of having to liter the code with all these warning disables on account of it, it looks like code smell.

Reply:
Yes, it's a pain that the Karambolo.PO library has so many non-compliant types. I don't think that CLS compliance is always essential - e.g. things like relying on case sensitivity, as I don't think anyone uses VB any more ;)  I will see if I can determine what the issue is and report back, but I think a PR for that library would likely be a step too far!